bin/exec.ml

```diff
@@ -95,7 +95,11 @@ module Command_to_exec = struct
     { get_path_and_build_if_necessary; prog; args; env }
     =
     get_path_and_build_if_necessary prog
-    |> Fiber.map ~f:(Result.map ~f:(spawn_process ~root ~args ~env))
+    |> Fiber.map
+         ~f:
+           (Result.map ~f:(fun prog ->
+              Console.reset ();
```

Comment:
Rather than doing this directly, let's be consistent with `dune build` watch mode.
```
              Scheduler.maybe_clear_screen ~details_hum:[] config;
```
You will need to thread `config` from `run_eagar_watch`.

Reply:
Thanks for this suggestions. This somehow fixed the issues in tests where the success messages was being truncated.

Reply:
Oh I see, because `maybe_clear_screen` has different behaviour when running inside the dune repo than when running in the wild.